C/plugins/common/get_plugin_info.cpp

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
This is in the wrong place - it is not common to plugins, it is a utility. Files in plugins/common is meant to be shared between multiple plugins.

Please add a plugins/utility directory for this.

Reply:
ok sure.